files/scripts/swss.sh

```diff
@@ -159,14 +161,28 @@ start() {
     fi
 
     # start service docker
-    /usr/bin/${SERVICE}.sh start $DEV
-    debug "Started ${SERVICE}$DEV service..."
+    if ! is_chassis_supervisor; then
```

Comment:
Add a comment as to why we are skipping this for chassis supervisor. Similar comment later in wait()

Reply:
Done